packages/core/data-transfer/src/strapi/providers/local-destination/strategies/restore/index.ts

```diff
@@ -33,12 +33,14 @@ export const deleteRecords = async (strapi: Strapi.Strapi, options: IRestoreOpti
 };
 
 const deleteEntitiesRecords = async (
-  strapi: Strapi.Strapi,
+  strapi: LoadedStrapi,
   options: IRestoreOptions = {}
 ): Promise<IDeleteResults> => {
   const { entities } = options;
   const query = queries.entity.createEntityQuery(strapi);
-  const contentTypes = Object.values<Schema.ContentType>(strapi.contentTypes);
+  const contentTypes = Object.values<Schema.ContentType>(
+    strapi.contentTypes as Record<string, Schema.ContentType>
```

Comment:
I'm confused why the assertion is necessary when strapi.contentTypes is typed with

`Shared.ContentTypes` which is `[key: UID.ContentType]: Schema.ContentType`

If we have to use the type assertion should it not be `Record<UID.ContentType, Schema.ContentType>`?

Reply:
This is because Object.values exect something with string keys and UID.ContentType is not just a string but a string pattern which isn't compatible unless I explicitly tell it.